schedule/kernel/run_kselftests.yaml

```diff
@@ -0,0 +1,7 @@
+name:          kselftests
+description:    >
+    Generic scheduler for Linux kselftests
+schedule:
+    - boot/boot_to_desktop
+    - kernel/run_kselftests
+    - shutdown/shutdown
```

Comment:
could you please copy bootloader section from kdump schedule to add support for other archs? or do you want to add later?